content/en/docs/setup/install/helm/index.md

```diff
@@ -175,7 +175,7 @@ to manage the lifecycle of Istio.
    If not already defined, create one using following command:
 
     {{< text bash >}}
-    $ kubectl apply -f @install/kubernetes/helm/helm-service-account.yaml@
+    $ kubectl apply -f @manifests/UPDATING-CHARTS.md@
```

Comment:
This doesn't look right ... how can an md file replace a yaml file?

Reply:
There is a follow on PR to update all the links https://github.com/istio/istio.io/pull/6458